lib/resty/mysql.lua

```diff
@@ -215,18 +231,129 @@ local function _dumphex(data)
 end
 
 
+local function _pwd_hash(password)
+    local add = 7
+
+    local hash1 = 1345345333
+    local hash2 = 0x12345671
+
+    local len = #password
+    for i = 1, len do
+        -- skip spaces and tabs in password
+        local tmp = strbyte(password, i)
+        local c = strchar(tmp)
+        if c ~= ' ' and c ~= '\t' then
```

Comment:
```suggestion
        if tmp ~= 32 and tmp ~= 9 then    -- " " and "\t"
```

we'd better avoid creating new Lua String.

Reply:
and `tmp` is not a good variable name, `byte` can be better.